configs/components/windows_ruby.json

```diff
@@ -1,7 +1,7 @@
 {
   "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet-win32-ruby.git",
   "ref": {
-    "x86": "refs/tags/2.1.8.2-x86",
-    "x64": "refs/tags/2.1.8.2-x64"
+    "x86": "refs/tags/2.1.8.1-x86",
+    "x64": "refs/tags/2.1.8.1-x64"
```

Comment:
@Iristyle changed per your comment in https://tickets.puppetlabs.com/browse/PA-205